### ISSUE TITLE
DAG not found in serialized table

### ISSUE THREAD

**Issue by Nnn7bb** [apache/airflow#14136]
I am still seeing a lot of errors in 2.0.1rc2 like

airflow.exceptions.SerializedDagNotFound: DAG 'targeting_pipeline_dappack.shard_3' not found in serialized_
dag table

Is this not part of rc2? Or is it likely a different problem?

_Originally posted by @fjmacagno in https://github.com/apache/airflow/issues/13893#issuecomment-775310886_

**Comment by ashb** [comment]
Could you try and give us reproduction steps @Nnn7bb please?

**Comment by damon09273** [comment]
@ashb I'm use 2.0.1, but I get a lot of same error too.

**Comment by ashb** [comment]
@damon09273 We'll need reproduction steps then please. How/when does this error come up?

**Comment by damon09273** [comment]
@ashb It happens in the staging environment, but didn't in the local environment, I use same Image.
![image](https://user-images.githubusercontent.com/17487840/110416934-8d2c2c00-80cf-11eb-8c33-9d911adbf1fe.png)

**Comment by danarwix** [comment]
Hi, 
Just got the same error while performing the astronomer certification.

steps to reproduce:

1. install the astronomer [cli ](https://www.astronomer.io/docs/cloud/stable/develop/cli-quickstart)
2. open a terminal
3. run: `mkdir <your directory name>`
4. run: `astro dev init`
5. open the `Dockerfile` and replace its content with ```bash FROM quay.io/astronomer/ap-airflow:2.0.0-2-buster-onbuild```
6. open the airflow UI at `http:// localhost:8080`
7. open the `example_dag`

expected result:
the DAG should be visible

actual result:
The following error is shown:
```python
Traceback (most recent call last):
File “/usr/local/lib/python3.7/site-packages/flask/app.py”, line 2447, in wsgi_app
response = self.full_dispatch_request()
File “/usr/local/lib/python3.7/site-packages/flask/app.py”, line 1952, in full_dispatch_request
rv = self.handle_user_exception(e)
File “/usr/local/lib/python3.7/site-packages/flask/app.py”, line 1821, in handle_user_exception
reraise(exc_type, exc_value, tb)
File “/usr/local/lib/python3.7/site-packages/flask/_compat.py”, line 39, in reraise
raise value
File “/usr/local/lib/python3.7/site-packages/flask/app.py”, line 1950, in full_dispatch_request
rv = self.dispatch_request()
File “/usr/local/lib/python3.7/site-packages/flask/app.py”, line 1936, in dispatch_request
return self.view_functionsrule.endpoint
File “/usr/local/lib/python3.7/site-packages/airflow/www/auth.py”, line 34, in decorated
return func(*args, **kwargs)
File “/usr/local/lib/python3.7/site-packages/airflow/www/decorators.py”, line 97, in view_func
return f(*args, **kwargs)
File “/usr/local/lib/python3.7/site-packages/airflow/www/decorators.py”, line 60, in wrapper
return f(*args, **kwargs)
File “/usr/local/lib/python3.7/site-packages/airflow/www/views.py”, line 1861, in tree
dag = current_app.dag_bag.get_dag(dag_id)
File “/usr/local/lib/python3.7/site-packages/airflow/utils/session.py”, line 65, in wrapper
return func(*args, session=session, **kwargs)
File “/usr/local/lib/python3.7/site-packages/airflow/models/dagbag.py”, line 171, in get_dag
self._add_dag_from_db(dag_id=dag_id, session=session)
File “/usr/local/lib/python3.7/site-packages/airflow/models/dagbag.py”, line 227, in _add_dag_from_db
raise SerializedDagNotFound(f"DAG ‘{dag_id}’ not found in serialized_dag table")
airflow.exceptions.SerializedDagNotFound: DAG ‘example_dag’ not found in serialized_dag table

```

**Comment by ashb** [comment]
@danarwix Could you try with `2.0.0-3-buster-onbuild` please? -- that has the fix backported from 2.0.1 for this issue.

**Comment by danarwix** [comment]
> @danarwix Could you try with `2.0.0-3-buster-onbuild` please? -- that has the fix backported from 2.0.1 for this issue.

@ashb that most certainly did the trick 😺

**Comment by ashb** [comment]
Marking this as fixed then!
